Automattic/newspack-blocks
# Add Newspack Logo to block picker.

### All Submissions:






### Changes proposed in this Pull Request:

Add the Newspack logo to the block picker, as pictured here:

<img width="420" alt="Screen Shot 2019-06-13 at 9 45 00 AM" src="https://user-images.githubusercontent.com/1477002/59437633-ef9fad80-8dbf-11e9-99e9-a98b1ec0501a.png">

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Create a new Post or Page, and click the + icon to add a block.
3. Scroll to the bottom of the block picker, and verify that the Newspack logo appears to the right of the Newspack category name.

### Other information:
